server/utils/utils.go

```diff
@@ -55,3 +55,11 @@ func Map(vs []string, f func(string) string) []string {
 	}
 	return vsm
 }
+
+func IsJiraCloudURL(jiraURL string) (bool, error) {
+	u, err := url.Parse(jiraURL)
+	if err != nil {
+		return false, err
+	}
+	return strings.HasSuffix(u.Hostname(), ".atlassian.net"), nil
```

Comment:
I was testing inputs like `test.atlassial.net'`, with a ' at the end. `.Hostname()` includes the ' and therefore it interprets the url as a Jira Server url (since it doesn't end with ".atlassian.net"). This is unfortunate, but acceptable I guess. We could introduce a host name syntax checker, but that would be a different task.